Signal/src/ViewControllers/ContactsViewHelper.h

```diff
@@ -7,15 +7,24 @@ NS_ASSUME_NONNULL_BEGIN
 @class ContactsViewHelper;
 @class Contact;
 @class SignalAccount;
+@protocol CNContactViewControllerDelegate;
 
 @protocol ContactsViewHelperDelegate <NSObject>
 
 - (void)contactsViewHelperDidUpdateContacts;
 
+@optional
+
 - (BOOL)shouldHideLocalNumber;
 
 @end
 
+@protocol ContactEditingDelegate <CNContactViewControllerDelegate>
+
+- (void)didFinishEditingContact;
```

Comment:
Could just be @optional method of `ContactsViewHelperDelegate`, but this works too.